src/Core/src/Handlers/Image/ImageHandler.cs

```diff
@@ -21,6 +21,10 @@ public partial class ImageHandler : IImageHandler
 #if __ANDROID__ || WINDOWS || TIZEN
 			[nameof(IImage.Background)] = MapBackground,
 #endif
+			[nameof(IImage.Height)] = MapHeight,
+			[nameof(IImage.Width)] = MapWidth,
+			[nameof(IImage.MaximumHeight)] = MapMaximumHeight,
```

Comment:
If this affects only windows, should we rather add the mappers in a `#if WINDOWS` block as this code is really overriding the code for all platforms.